code/game/objects/items/weapons/maneki_neko.dm

```diff
@@ -33,7 +33,6 @@
 	..()
 
 /obj/item/weapon/maneki_neko/Process()
-	..()
```

Comment:
for what purpose?

Reply:
..() end Process before it even start.

Reply:
That'd only happen if you returned the parent call's result, but fair enough.